Application/EdFi.Admin.DataAccess/Repositories/ClientAppRepo.cs

```diff
@@ -5,8 +5,7 @@
 
 using System;
 using System.Collections.Generic;
-using System.Data.Entity;
-using System.Data.Entity.Migrations;
+using Microsoft.EntityFrameworkCore;
```

Comment:
Reorder using statements to get System.* on top and the rest of the libraries in alphabetic order.